### PR TITLE
Part2

### PR DESCRIPTION
**Pull request by SChandarana**
Part 2 Complete

### CODE REVIEW

**Review by FridaTveit — COMMENTED** [review]
Since `part2` has been branched off `part1` this pull request contains all the changes from your first pull request as well. There are a few ways to fix that:

1. Do markups on your first pull request before putting up a second. This is the preferred option as it will cause the least amount of confusion. You should also generally prioritise things already in review over new work, as that means things will go quicker through review and will be easier to review (as the reviewer gets to see the changes quicker) 🙂 

2. Change the target branch to `part1`. I sometimes do this if I have some new changes I want to put up for review and I'm still waiting for the first changes to be reviewed. In that case I would open a PR with `part1` as the target branch but NOT MERGE IT until the `part1` PR has been merged (because I don't really want to merge it into that branch). Once the first PR has been merged I'd change the target branch of the second PR to `master` and then merge it (once it's been approved) 🙂

**Comment by SChandarana** [comment]
Just testing some things out. Once part1 branch is complete we can merge to main and see what happens here

File: FizzBuzz/Program.cs
Comment (FridaTveit) [inline FizzBuzz/Program.cs]: Is this comment really useful? You should only need to add comments if it's something someone will need to know when they're reading the code for the first time. And you should always consider whether it's possible to refactor the code to make it more readable instead of adding a comment. Ideally your code should be so neat and clear that no comments are necessary 🙂

File: FizzBuzz/Program.cs
Comment (FridaTveit) [inline FizzBuzz/Program.cs]: If this is a comment specifically for me then you can add a comment to the pull request. That way you don't need to remember to remove the comment later 🙂 

And yes, I would recommend using a list instead as your solution has gotten quite complicated! Using a list should make it neater and easier to understand 🙂

**Review by FridaTveit — COMMENTED** [review]
A good start! I've left some suggestions for improvements 🙂

File: FizzBuzz/Program.cs
Comment (FridaTveit) [inline FizzBuzz/Program.cs]: You can use `var` here instead of `List<string>` 🙂

File: FizzBuzz/Program.cs
Comment (FridaTveit) [inline FizzBuzz/Program.cs]: Could you get the index of the first B word directly, instead of getting the word and then finding the index for it? 🙂

File: FizzBuzz/Program.cs
Comment (FridaTveit) [inline FizzBuzz/Program.cs]: Very minor but you've got an uneeded extra empty line here 🙂

File: FizzBuzz/Program.cs
Comment (FridaTveit) [inline FizzBuzz/Program.cs]: Uneeded empty line 🙂

File: FizzBuzz/Program.cs
Comment (FridaTveit) [inline FizzBuzz/Program.cs]: Could you simplify this so that you only need one `Console.Write` statement? 🙂

**Review by FridaTveit — COMMENTED** [review]
A few minor suggestions, otherwise looks good! 🙂

**Review by FridaTveit — APPROVED** [review]
Looks great 🙂

**Pull request merged**
